diff --git a/module/index.js b/module/index.js
index 1234567..abcdefg 100644
--- a/module/index.js
+++ b/module/index.js
@@ -67,7 +67,7 @@ const executeOnFiles = async ({ filePathList, config, options, }) => {
     const results = await pMap(filePathList, mapper, {
         // Avoid: EMFILE: too many open files, uv_cwd
         // https://github.com/secretlint/secretlint/issues/72
-        concurrency: os.cpus().length,
+        concurrency: Math.max(os.cpus().length, 1),
     });
     debug("executeOnFiles result counts: %s", results.length);
     secretLintProfiler.mark({
